stubs/protobuf/google/protobuf/internal/type_checkers.pyi

```diff
@@ -0,0 +1,15 @@
+from typing import Generic, Protocol, TypeVar
+
+_T = TypeVar("_T")
+
+class _ValueChecker(Protocol[_T]):
```

Comment:
I put an underscore here since the protocol is not an actual element exposed within this module.